### ISSUE TITLE
No such file or directory: '/var/cache/swift/object.recon'

### ISSUE THREAD

**Issue by shooding** [NVIDIA/docker-swift#7]
I managed to find a fix here:
https://review.openstack.org/#/c/9798/4/doc/source/development_saio.rst

Please add the following 

```
mkdir -p /var/cache/swift /var/cache/swift2 /var/cache/swift3 /var/cache/swift4
chown swift:swift /var/cache/swift*
```

**Comment by kahing** [comment]
where do you see this error?

**Comment by timuralp** [comment]
Fixed via 3ab3f13e62b7d96f9b242694840a5a246e420c7c
